guides/define_custom_kernel.py

```diff
@@ -0,0 +1,412 @@
+"""
```

Comment:
You also have to render the guide by running

```
cd scripts
python autogen.py add_guide define_custom_kernel
```
and add the other files to the PR. You'll also need to add this guide to the index:

https://github.com/keras-team/keras-io/blob/80982a533b4e69141deb849047b639303b54c6f1/scripts/guides_master.py#L43

Reply:
Oh I didn't even realized that's the recommend flow.

Yes, you need a TPU VM to really render it. There is a way to do it in colab too, although it's a bit contrived.

Reply:
I'm a bit confused about what the CPU rendering did because I don't see the output blocks.